schema.sql

```diff
@@ -6,6 +6,7 @@ CREATE TABLE animals (
     date_of_birth DATE NOT NULL,
     escape_attempts INTEGER NOT NULL,
     neutered BOOLEAN NOT NULL,
-    weight_kg DECIMAL NOT NULL
+    weight_kg DECIMAL NOT NULL,
+    species VARCHAR(100)
```

Comment:
- [ ] Please do not manually alter the table like you have when making changes. When working with SQL in the future, manually changing the data in your table can lead to issues in your database as you shall see. Instead, add SQL code that can alter the table for example:
```sql    
ALTER TABLE animals ADD species varchar(100);
 ```